src/effects/lv2/LV2Effect.cpp

```diff
@@ -99,12 +102,13 @@ BEGIN_EVENT_TABLE(LV2EffectMeter, wxWindow)
    EVT_SIZE(LV2EffectMeter::OnSize)
 END_EVENT_TABLE()
 
-LV2EffectMeter::LV2EffectMeter(wxWindow *parent, const LV2ControlPortPtr port)
-:  wxWindow(parent, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxDEFAULT_CONTROL_BORDER),
-   mControlPort(port)
+LV2EffectMeter::LV2EffectMeter(
+   wxWindow *parent, const LV2ControlPortPtr port, const float &value
+)  : wxWindow(parent, wxID_ANY, wxDefaultPosition, wxDefaultSize,
+      wxDEFAULT_CONTROL_BORDER), mControlPort(port)
+   , mValue{ value }
```

Comment:
We can drop `const float& value` I think and just initialize `mValue{ port->mVal }` probably.

I've spent some time to convince myself that the lifetime of the pointers (references) match.  But I haven't seen next commits, probably they justify the need for the third argument.

Reply:
Yep, they do justify

Reply:
But, as you must understand by now, this leads up to removal of `mVal` from the port structure so it too is an immutable